ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -60,17 +62,27 @@
   @Autowired private MockMvc mvc;
   @Autowired private RestTemplate restTemplate;
 
-  private MockRestServiceServer server;
+  @Autowired
+  @Qualifier("nonBufferingRestTemplate")
+  private RestTemplate nonBufferingRestTemplate;
+
+  private MockRestServiceServer storeServer;
+  private MockRestServiceServer transformServer;
 
   @Before
   public void beforeEach() {
-    server = MockRestServiceServer.createServer(restTemplate);
+    MockRestServiceServerBuilder builder = MockRestServiceServer.bindTo(nonBufferingRestTemplate);
+    builder.bufferContent();
+    storeServer = builder.build();
+    transformServer = MockRestServiceServer.createServer(restTemplate);
```

Comment:
❓Why can't you create the `storeServer` the same way as the `transformServer`?

```java
@Before
public void beforeEach() {
  storeServer = MockRestServiceServer.createServer(nonBufferingRestTemplate);
  transformServer = MockRestServiceServer.createServer(restTemplate);
}
```

Reply:
That might be left over from something I tried originally. Let me take it out and see it the tests still pass.